package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Case Western Reserve's hackathon website.",
   "main": "index.js",
   "scripts": {
-    "build": "babel src/ -d lib/ && NODE_ENV=development webpack --config webpack/webpack.config.js",
```

Comment:
Note for jack: This shouldn't be included in the merge, this was the windows patch for the way you defined environment variables